dev/create-release/release-build.sh

```diff
@@ -181,10 +183,6 @@ if [[ "$1" == "package" ]]; then
   rm spark-$SPARK_VERSION/NOTICE-binary
   rm -r spark-$SPARK_VERSION/licenses-binary
   tar cvzf spark-$SPARK_VERSION.tgz spark-$SPARK_VERSION
-  echo $GPG_PASSPHRASE | $GPG --passphrase-fd 0 --armour --output spark-$SPARK_VERSION.tgz.asc \
-    --detach-sig spark-$SPARK_VERSION.tgz
-  echo $GPG_PASSPHRASE | $GPG --passphrase-fd 0 --print-md \
-    SHA512 spark-$SPARK_VERSION.tgz > spark-$SPARK_VERSION.tgz.sha512
```

Comment:
Ur, this is still used during the official release process, isn't it? What about adding option to skip this part?
`do-release-docker.sh` -> `do-release.sh` -> `release-build.sh package`
cc @vanzin

Reply:
maybe a better idea is to make singing a separated phase. The `do-release.sh` should first call `release-build.sh package` to get the binary files, and call `release-build.sh sign` to sign them.

Reply:
yeah, this PR would break everything.  i opened up a new one w/better logic here:  https://github.com/apache/spark/pull/23492

the release tooling scripts are a nightmare...